crutest/src/main.rs

```diff
@@ -1344,11 +1344,16 @@ async fn main() -> Result<()> {
             )
             .await?;
         }
-        Workload::ReplaceBeforeActive {
-            dsc_str,
-            replacement,
-        } => {
-            let dsc_client = Client::new(&dsc_str);
+        Workload::ReplaceBeforeActive { replacement } => {
+            let dsc_client = match opt.dsc {
+                Some(dsc_addr) => {
```

Comment:
This is a small behavior change: previously, there was a default value of `http://127.0.0.1:9998`; now, we return an error.  Dunno if the change is intentional or not!

Reply:
Yeah, before it assumed that any `dsc` it finds at the default would be correct, but that
may not be what the user wanted.  More so now that there is a global `dsc` option which
should be what these tests use.  Hopefully fewer surprises as this makes the dsc server
target required.